plaso/parsers/sqlite_plugins/windows_eventtranscript.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+"""SQLite parser plugin for Windows Diagnosis EventTranscript database file."""
+import json
+
+from dfdatetime import filetime as dfdatetime_filetime
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+
+class EventTranscriptEventData(events.EventData):
+  """Windows diagnosis EventTranscript event data.
+
+  Attributes:
+    sid (str): Windows Security ID of a user account.
```

Comment:
SID is the type but what is this used for? I assume it is used to identify the user